### PR TITLE
docs: render docs for examples

### PR DESCRIPTION
**Pull request by Duologic**
This PR adds a page explaining two of the examples. This should make it more accessible to
a wider audience.

### CODE REVIEW

File: grafonnet-base/veneer/panel.libsonnet [hunk 1]
```diff
@@ -149,7 +149,7 @@ function(name, panel)
     fieldOverride:
       local matchers = [
         'byName',
-        'byRegex',
+        'byRegexp',
```
Comment (Duologic) [inline grafonnet-base/veneer/panel.libsonnet:152]: drive-by fix

**Review by malcolmholmes — APPROVED** [review]
